functions/go/set-setter/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM golang:1.13-stretch
```

Comment:
We have decided to use a specific version of alpine. Please see other Go functions' Dockerfile.